### DISCUSSION TITLE
What information is required for the GitHub Education application?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54362] [GitHub Education]
### Hi everyone,

Hello.
Not to brag, but I have had my application denied 34 times.
Here is what I tried

- I used the school's email address (ed.jp).
- I am Japanese, so I added an English translation next to the Japanese.
- I uploaded a clean student ID card with no English translation.
- I wrote the translations together on the outside of the student ID card, not inside.
- I uploaded the student ID card and the translation in two separate uploads.
- I filled out my profile properly.
- I re-logged in and then submitted my application.
- I tried smart phones, PCs, browsers, and incognito mode.

I have done these things and still only receive the following message.

Hey (MYNAME)
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 3, 2023 for @(MYNAME)

Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved. 
Once you have resolved the issues with your sumbission, visit us again and reapply. 
Have questions? Join the GitHub Education Community for answers. The GitHub Education Team

What is the real reason this message is being sent?

**Comment by alfi-dim** [comment]
hey! i have same problem, I've been rejected 12 times, and i tried every document and proof i have, still no luck, though it sad that my university not provide academic email (I'm from Indonesia btw), i hope we can get the benefit, don't lose hope my friend 👍

**Reply by ghost** [reply]
Hi, I'm a little relieved to know that you are in the same situation and that I'm not the only one.
I have looked at the internet and many of the other discussions, but in the end none of them were a solution to me.
Perhaps I am missing a condition that others naturally meet.
The only thing I can say is that it is a waste of time, so don't apply until someone here comes up with a solution.🫤

**Reply by alfi-dim** [reply]
Soo uhh... Did you manage to get it?

**Reply by ghost** [reply]
No, not yet. I have half given up, but will continue to gather information.🧐
